docs/source/index.rst

```diff
@@ -1,10 +1,9 @@
-.. Metadrive documentation master file, created by
-   sphinx-quickstart on Sun Apr 28 22:26:41 2019.
-   You can adapt this file completely to your liking, but it should at least
-   contain the root `toctree` directive.
+Metadrive: Unified API for working with different kinds of resources
+===================================================================
 
-Welcome to Metadrive's documentation!
-=====================================
+**Metadrive** is a unified API for gathering information from different kinds of resources.
```

Comment:
> a unified API for gathering information from different kinds of resources.

Actually, not only gathering, but also managing.. Not just input, but also output.

May be better along the lines of:
a unified API for managing different kinds of resources (gathering information and performing operations).

Reply:
I guess that the common word `managing` is a bit confusing. What about the following:

> a unified API for gathering information from and performing operations on different kinds of resources.

Reply:
@eugulixes yes, that is good way to put it!